src/Cli/dotnet/commands/dotnet-sln/list/SlnListReportOutputFormat.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Tools.Sln.List
+{
+    internal enum SlnListReportOutputFormat
+    {
+        console_with_header = 0,
```

Comment:
I think what we should do here is to only have `text` and `json` layouts. Looking at other commands we have the following formats:

* table (not applicable here)
* json (applicable here)

In previous issues we've discussed using `text` as the name for not-uniformly formatted, non-tabular output so I think we should use that here.

I also don't think that there should be a separate 'without header' option. The direction we have been taking in the `dotnet` CLI for user-parseable output is "if a user wants structured output they should use JSON output and manipulate it with tools like `jq`".

Reply:
Applied this change